service/messages/messages_replies.json

```diff
@@ -14,19 +17,22 @@
         "hey %s! What can i help you with?",
         "hey %s!",
         "hi %s!",
-        "hello %s!"
+        "hello %s!",
+        "hey %s! I am ready here. Anything I may help you with?"
     ],
     "help": [
-        "I can do lots of things! You can ask for web search, specific google/yahoo/bing search, images, videos and also find answers for any of your queries!",
-        "You can ask for web search, specific google/yahoo/bing search, images, videos and also find answers for any of your queries!",
-        "I can help you with web search, specific google/yahoo/bing search, images, videos and also find answers for any of your queries!",
-        "My work is to help you with the web search, specific google/yahoo/bing search, images, videos and also find answers for any of your queries!"
+        "I can do lots of things! You can ask for web search, specific google/yahoo/bing search, images, videos, search for medicines specifications, set reminder and also find answers for any of your queries!",
+        "You can ask for web search, specific google/yahoo/bing search, images, videos, search for medicines specifications, set reminder and also find answers for any of your queries!",
+        "I can help you with web search, specific google/yahoo/bing search, images, videos, search for medicines specifications, set reminder and also find answers for any of your queries!",
+        "My work is to help you with the web search, specific google/yahoo/bing search, images, videos, search for medicines specifications, set reminder and also find answers for any of your queries!"
```

Comment:
also include ***tasklist*** after ***set-reminders*** since this feature would be added soon